### PR TITLE
Release candidate changes

### PR DESCRIPTION
**Pull request by lbarbon**

### CODE REVIEW

File: src/valiant/metadata_utils.py [hunk 1]
```diff
@@ -74,32 +107,36 @@ def get_slice(s: str, a: int, b: int) -> str:
...
     return [
         get_slice(s, a, b)
         for s, (a, b) in zip(
-            df.loc[mask, string_col_name],
+            _get_field(df, string_col_name, mask=mask),
             zip(
-                df.loc[mask, start_col_name],
-                df.loc[mask, end_col_name]
+                _get_field(df, start_col_name, mask=mask),
+                _get_field(df, end_col_name, mask=mask)
```
Comment (FMakosza) [inline src/valiant/metadata_utils.py:121]: Could this be simplified down to the following?
```python
for s, a, b in zip(
    _get_field(df, string_col_name, mask=mask),
    _get_field(df, start_col_name, mask=mask),
    _get_field(df, end_col_name, mask=mask))
```

File: src/valiant/models/targeton.py [hunk 2]
```diff
@@ -611,34 +667,29 @@ def get_pam_ext_ref_end(self, pos: int) -> int:
         )
 
     def get_sgrna_ids(self, spr: StrandedPositionRange) -> FrozenSet[str]:
-        return frozenset({
+        cds = frozenset({
             self._codon_to_pam_variant[codon_index].sgrna_id
```
Comment (FMakosza) [inline src/valiant/models/targeton.py:671]: Method definition seems to be missing?

Comment (lbarbon) [reply src/valiant/models/targeton.py:671]: The `_codon_to_pam_variant` attribute is set via `object.__setattr__`, so static analysis may fail to recognise it.

File: src/valiant/metadata_utils.py (hunk 1 above)
Comment (lbarbon) [reply src/valiant/metadata_utils.py:121]: It could and should, yes.

**Pull request closed without merging**